### DISCUSSION TITLE
请问设置怎么保存？

### DISCUSSION THREAD

**Discussion by Quakerhkjp** [dyphire/mpv-config#36] [Q&A] [answered]
老大，每次设置好了，重启就又恢复到默认了，有保存设置的方法吗？

**Comment by dyphire** [comment, marked as answer]
可以在 [persist_properties.conf](https://github.com/dyphire/mpv-config/blob/master/script-opts/persist_properties.conf) 中设置需要保存全局状态的选项

另外 `--reset-on-next-file` 选项中的参数会在播放下一文件时重置，视需要自行删增
https://github.com/dyphire/mpv-config/blob/1aa8ddc6b0a6b5d81521e446862fc93edc2ef55e/mpv.conf#L111-L112
